### PR TITLE
Stats: Number of unit admins

### PR DESCRIPTION
**Pull request by i-oden**
## Before submitting this PR

1. **Description:** Save number of unit admins to database, in the Reporting table.
2. **Jira task / GitHub issue:** DDS-1551
3. **How to test:** 
    1. In the dds_web container, run `flask db upgrade` and `flask db downgrade` to verify the migrations. 
    2. After one `flask db downgrade`, add a row manually to the Reporting table, and then run `flask db upgrade`. There should be no errors.
4. **Type of change:** [_Check the relevant boxes in the section below_](#what-type-of-changes-does-the-pr-contain)
5. **Add docstrings and comments to code**, _even if_ you personally think it's obvious.

## What _type of change(s)_ does the PR contain?

- [x] New feature
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [x] Non-breaking
- [x] Database change
  - [x] Migration _included in PR_
  - [ ] Migration _not needed_
- [ ] Bug fix
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [ ] Non-breaking
- [ ] Security Alert fix
- [ ] Documentation
- [ ] Tests **(only)**
- [ ] Workflow

## Checklist

- [Sprintlog](../SPRINTLOG.md)
  - [x] Added
  - [ ] Not needed (E.g. PR contains _only_ tests)
- Rebase / Update / Merge _from_ base branch (the branch from which the current is forked)
  - [ ] Done
  - [ ] Not needed
- Blocking PRs
  - [ ] Merged
  - [ ] No blocking PRs
- PR to `master` branch
  - [ ] Yes: Read [the release instructions](../doc/procedures/new_release.md)
    - [ ] I have followed steps 1-7.
  - [x] No

## Actions / Scans

- **Black**: Python code formatter. Does not execute. Only tests.
  Run `black .` locally to execute formatting.
  - [x] Passed
- **Prettier**: General code formatter. Our use case: MD and yaml mainly.
  Run `npx prettier --write .` locally to execute formatting.
  - [x] Passed
- **Yamllint**: Linting of yaml files.
  - [x] Passed
- **Tests**: Pytest to verify that functionality works as expected.
  - [ ] New tests added
  - [x] No new tests
  - [x] Passed
- **CodeQL**: Scan for security vulnerabilities, bugs, errors
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Trivy**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Snyk**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed

<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1419"></a>

### CODE REVIEW

File: dds_web/database/models.py [hunk 1]
```diff
@@ -1080,6 +1080,7 @@ class Reporting(db.Model):
     date = db.Column(db.DateTime(), unique=True, nullable=False, default=datetime.date.today)
     unit_count = db.Column(db.Integer, unique=False, nullable=False)
     researchuser_count = db.Column(db.Integer, unique=False, nullable=False)
-    unit_personnel_count = db.Column(db.Integer, unique=False, nullable=False)
+    unit_personnel_count = db.Column(db.Integer, unique=False, nullable=True)
```
Comment (i-oden) [inline dds_web/database/models.py:1083]:
```suggestion
    unit_personnel_count = db.Column(db.Integer, unique=False, nullable=False)
```

**Comment by i-oden** [comment]
@valyo This one is ready now, tests are fixed

**Comment by valyo** [comment]
Everything seems fine but following "Hot to test" doesn't behave as expected (or I am not doing it corretly)

**Comment by i-oden** [comment]
> Everything seems fine but following "Hot to test" doesn't behave as expected (or I am not doing it corretly)

How does it behave?

**Comment by valyo** [comment]
Actually it does

**Review by valyo — APPROVED** [review]

**Pull request merged**